.gitignore

```diff
@@ -1,3 +1,4 @@
+lib
```

Comment:
Because I'm using intellij's OSGI plugin and that insists on adding some jars there. Harmless, I think?